### ISSUE TITLE
How to skip some operation when doing QAT with aimet_torch?

### ISSUE THREAD

**Issue by charmway** [quic/aimet#4055]
Hello, we have found that some operators in ONNX suffer severe accuracy loss after QAT. 

Can we skip these operators during QAT training? Alternatively, after training generates the encoding, how can we skip quantization for these operators when using QNN for inference?

**Comment by charmway** [comment]
Hi, need some help~

**Comment by quic-kyunggeu** [comment]
Hi @charmway.

If you want to exclude certain operations from QAT, you can simply set those layers to `requires_grad=False`. For example, `sim.model.fc.requires_grad_(False)`

Once the encoding of a layer is already exported, there's no way you can "skip" quantization of those layers in QNN. If you wish to not quantize those layers, you shouldn't export the encodings of those layers. To achieve this, you can remove the quantizers by calling [remove_all_quantizers](https://quic.github.io/aimet-pages/releases/latest/apiref/torch/utils.html#aimet_torch.utils.remove_all_quantizers) on the layers you want to exclude. For example, `aimet_torch.utils.remove_all_quantizers(sim.model.fc)`

**Comment by charmway** [comment]
Get it. Tks~
